### PR TITLE
Add FAQ note on how to add/remove current env name from cmd prompt

### PR DESCRIPTION
**Pull request by electronwill**
Add note to FAQ on how to add or remove the name of the current active
evironment from the command prompt.

### CODE REVIEW

**Comment by electronwill** [comment]
@asmeurer @ilanschnell Ready to merge.

File: docs/source/faq.rst [hunk 1]
```diff
@@ -205,7 +205,36 @@ Getting info about environments
 
 SEE ALSO: the :doc:`/using/envs` section of our documentation site. 
 
-2. How can I list all installed packages in a specific environment, for example, ``myenv``?
+2. How can I show the name of the active environment in my command prompt?
+
+   The anaconda (or miniconda) installer asks whether it should always add the name 
+   of the active environment to the command prompt by inserting it into the environment 
+   variable called PATH in the Bash shell. The default is no.
```
Comment (asmeurer) [inline docs/source/faq.rst]: This is wrong? Setting the PATH is completely separate from the changeps1 option, which the installer doesn't do anything with. And setting the PATH defaults to no only on Linux.

File: docs/source/faq.rst [hunk 2]
```diff
@@ -205,7 +205,36 @@ Getting info about environments
...
+
+   To turn it on later:
+   
+     .. code-block:: bash
+        
+        source deactivate
```
Comment (asmeurer) [inline docs/source/faq.rst]: You don't have to deactivate to change the option.

File: docs/source/faq.rst (hunk 1 above)
Comment (electronwill) [inline docs/source/faq.rst]: Ah, I see what you mean. I'll fix that and resubmit.

File: docs/source/faq.rst (hunk 2 above)
Comment (electronwill) [inline docs/source/faq.rst]: When I set changeps1 false while in an environment, the name of the environment that was current then remained in my prompt as I switched around between different environments. When I set changeps1 true, it didn't change my prompt until after the next source activate or source deactivate command. This behavior does make perfect sense, but if we assume that users want to see no environment name whenever changeps1 is false, and the correct environment name whenever changeps1 is true, then they should run 'source deactivate' before changing changeps1.

**Pull request closed without merging**